src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  margin: 0;
+
+}
+
+.icon {
+  position: absolute;
+  display: block;
+  color: #687c94;
+  padding-left: 26px ;
+  text-align: center;
+}
+
+.form {
+  height: 70px;
+  margin: 20px 8px 8px;
+  margin-bottom: 0;
+  display: flex;
+  align-items: center;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:focus {
```

Comment:
`form` doesn't have `focus` state

Reply:
I must remove focus for form? Because when I removed, focus for input doesn't work corectly.